uber/models.py

```diff
@@ -966,23 +1000,41 @@ class Room(MagModel, NightsMixin):
     department = Column(Choice(JOB_LOCATION_OPTS))
     notes      = Column(UnicodeText)
     nights     = Column(MultiChoice(NIGHT_OPTS))
```

Comment:
Note to self: Anthrocon doesn't use this table, so no migration is needed.  Since they have no data in this table, if they really wanted to play around with this, they could literally do a ``DROP TABLE room;`` and let the codebase re-create the table.